windows/gui-source/Kolibri/Kolibri.cpp

```diff
@@ -217,6 +217,10 @@ int WINAPI WinMain(HINSTANCE hInstance, HINSTANCE hPrevInstance, LPSTR lpCmdLine
 	if (dwMutexWaitResult != WAIT_OBJECT_0)
 	{
 		MessageBox(HWND_DESKTOP, getStr(ID_STRING_12_en), getStr(ID_STRING_13_en), MB_OK | MB_ICONINFORMATION);
+		if (isServerOnline("Kolibri session", "http://127.0.0.1:8080/learn/#!/learn"))

```

Comment:
Let's link to `http://127.0.0.1:8080/` rather than hard-coding a path.

We should update other instances too:

* https://github.com/learningequality/kolibri-installer-windows/pull/63/files#diff-03d5f197d7e34c30938a1baf6746d5daR115
* https://github.com/learningequality/kolibri-installer-windows/pull/63/files#diff-03d5f197d7e34c30938a1baf6746d5daR187

Reply:
Also, is it not possible to link to the URL generated by the status command instead of hardcoding this? Just in case someone has changed the default port or somesuch?

Reply:
> Also, is it not possible to link to the URL generated by the status command instead of hardcoding this? Just in case someone has changed the default port or somesuch?

Created a new issue for that here: https://github.com/learningequality/kolibri-installer-windows/issues/65

As far as I know though, we give no documented way of changing the port in Windows so I don't think this is a high priority?

Reply:
> As far as I know though, we give no documented way of changing the port in Windows

There's nothing impeding Windows users to start Kolibri with the "standard" way [we do document](http://kolibri.readthedocs.io/en/latest/manage.html#run-kolibri-from-a-different-port):

![virtualbox_win10_11_04_2018_22_49_18](https://user-images.githubusercontent.com/1457929/38659350-0259ec20-3dde-11e8-80f5-eef7982f0927.png)

Granted they will not be able to use the shortcut and taskbar launcher, but still...